src/sentry/middleware/auth.py

```diff
@@ -2,6 +2,18 @@
 
 from django.contrib.auth import middleware
 
+from sentry.utils.linksign import process_signature
+
 
 class AuthenticationMiddleware(middleware.AuthenticationMiddleware):
```

Comment:
Not related to this patch specifically, but we don't actually have this in the `MIDDLEWARE_CLASSES` for `getsentry`, though it is in the defaults for `sentry`. It might make sense to check in the link generation method that the class is actually in the middleware stack and log a warning if it's not, although I'm not quite sure the best way to do that.